daemon_windows.go

```diff
@@ -6,10 +6,13 @@
 package daemon
 
 import (
+	"fmt"
+	"os/exec"
 	"errors"
+	"github.com/kardianos/osext"
 )
 
-var ErrWindowsUnsupported = errors.New("windows daemon is not supported")
+var ErrWindowsUnsupported = errors.New("Adding as a service failed. Download and place nssm.exe in the path to install this service as an service. NSSM url: https://nssm.cc/")
```

Comment:
I have tried adding executable as a service with `sc.exe`, but it didn't work. Then I have tried `nssm`, and it worked like a charm. So I have added windows support with `nssm`. I have to try it now at in windows.